Basic-Car-Maintenance/Shared/Settings/Views/SettingsView.swift

```diff
@@ -65,6 +67,19 @@ struct SettingsView: View {
                             
                             Text(vehicle.model)
                         }
+                        .swipeActions {
+                            Button("Delete") {
+                              Task {
+                                do {
+                                  try await viewModel.deleteVehicle(vehicle)
+                                } catch {
+                                  errorDetails = error
+                                  showDeleteVehicleError = true
+                                }
+                              }
+                            }
+                            .tint(.red)
```

Comment:
Note that you don't need to set the `.tint` to red because you can instead give the Button `role: .destructive`